src/app/photos/components/Preview.tsx

```diff
@@ -30,43 +30,59 @@ export default function Preview({
   const [thumbnailSrc, setThumbnailSrc] = useState<string | null>(null);
 
   useEffect(() => {
-    if (previewIndex !== null && bucketId) {
-      setThumbnailSrc(null);
-
-      const photo = items[previewIndex];
-      getPhotoPreview({
-        photo,
-        bucketId,
-      }).then(setThumbnailSrc);
-    }
+    if (previewIndex === null) return;
+    setThumbnailSrc(null);
+    const photo = items[previewIndex];
+    const photoBucketId = photo.networkBucketId ? photo.networkBucketId : bucketId;
+    if (!photoBucketId) return;
+    getPhotoPreview({
+      photo,
+      bucketId: photoBucketId,
+    })
+      .then(setThumbnailSrc)
+      .catch((err) => {
+        Sentry.captureException(err, {
+          extra: {
+            photoId: photo.id,
+            bucketId: photoBucketId,
+          },
+        });
+      });
   }, [previewIndex, items]);
 
   const [src, setSrc] = useState<string | null>(null);
   const [itemType, setItemType] = useState<PhotosItemType>(PhotosItemType.PHOTO);
 
   useEffect(() => {
-    if (previewIndex !== null && bucketId) {
-      setSrc(null);
+    if (previewIndex === null) return;
 
-      const abortController = new AbortController();
-      const photo = items[previewIndex];
-      getPhotoBlob({ photo, bucketId, abortController })
-        .then((blob) => {
-          setItemType(photo.itemType);
-          return setSrc(URL.createObjectURL(blob));
-        })
-        .catch((err) => {
-          if (abortController.signal.aborted) {
-            return;
-          }
+    setSrc(null);
 
-          console.log(err);
+    const abortController = new AbortController();
+    const photo = items[previewIndex];
+    const photoBucketId = photo.networkBucketId ? photo.networkBucketId : bucketId;
+    if (!photoBucketId) return;
+
+    getPhotoBlob({ photo, bucketId: photoBucketId, abortController })
+      .then((blob) => {
+        setItemType(photo.itemType);
+        return setSrc(URL.createObjectURL(blob));
+      })
+      .catch((err) => {
+        Sentry.captureException(err, {
```

Comment:
avoid capturing aborted downloads moving the if before capturing exceptions on Sentry